integration-tests/src/test_loop/tests/in_memory_tries.rs

```diff
@@ -47,6 +46,7 @@ fn test_load_memtrie_after_empty_chunks() {
         genesis_builder.add_user_account_simple(account.clone(), initial_balance);
     }
     let (genesis, epoch_config_store) = genesis_builder.build();
+    let shard_layout = genesis.config.shard_layout.clone();
```

Comment:
So here we take the genesis shard layout and remove the `let tip = clients[idx].chain.head().unwrap(); let shard_layout = clients[idx].epoch_manager.get_shard_layout(&tip.epoch_id).unwrap();` below. Maybe it would be more sound to keep that part (replacing `idx` with `0`) where we find the current shard layout after the `test_loop.run_until()`, and move it up to right before the `let shard_uid = shard_layout.shard_uids().next().unwrap();` ?